docs/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/is_hvac_sys_heating_type_furnace.md

```diff
@@ -7,16 +7,13 @@
 - **hvac_b.id**: The id of the hvac system to evaluate.  
```

Comment:
I think the Description should be updated also to remove "or if it has more than 1 heating system".